### PR TITLE
First pass at Draw & Measure

### PR DESCRIPTION
**Pull request by PatchesMaps**

### CODE REVIEW

**Comment by glenselle** [comment]
Translations are being done here: https://github.com/MonsantoCo/ol-kit/pull/16

File: src/index.js [hunk 1]
```diff
@@ -1,2 +1,6 @@
 export { default as Controls, Compass, ZoomControls } from './Controls'
+export { default as BasemapManager, BingMaps, BlankWhite, OpenStreetMap, StamenTerrain, StamenTonerDark, StamenTonerLite } from './Basemaps'
+export { default as Draw, Box, Circle, Freehand, Line, Point, Polygon } from './Draw'
```
Comment (glenselle) [inline src/index.js]: Should these be namespaced? I feel like exporting `Point` is a bad idea -- what does that even mean? And what if we have another component related to points in the future. If we did `DrawPoint` then at least its more apparent what it is. Same for the basemaps: `BlankWhite` -> `BasemapBlankWhite` or `BlankWhiteBasemap` ?

File: src/index.js [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+export { default as Measure } from './Measure'
+export { default as Toolbar } from './Toolbar'
```
Comment (glenselle) [inline src/index.js]: This should probably be `DrawToolbar` unless its truly a general purpose toolbar.

File: src/index.js (hunk 1 above)
Comment (webbhead777) [reply src/index.js]: contextualizing is a good idea- just make sure to add a note that this is breaking from pre-release `0.0.0-0`

Comment (stazrad) [reply src/index.js:3]: namespacing has been merged to master

**Comment by PizzaBrandon** [comment]
Closing. We'll come back with refreshed code later.

**Pull request closed without merging**